wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/PrunerTest.kt

```diff
@@ -1750,7 +1750,33 @@ class PrunerTest {
     )
 
     val extension = pruned.protoFile("extension.proto")!!
-    assertThat(extension.imports).containsExactly("title.proto")
+    assertThat(extension.imports).containsExactly("message.proto", "title.proto")
```

Comment:
This was actually not producing a valid proto file after the pruning either as it had removed the required import for the message being extended.